src/sitewiseCache.ts

```diff
@@ -152,7 +153,13 @@ export class SitewiseCache {
   }
 
   async getAssetPickerOptions(): Promise<Array<SelectableValue<string>>> {
-    const options: Array<SelectableValue<string>> = [];
+    const options: Array<SelectableValue<string>> = getTemplateSrv()
+      .getVariables()
+      .map((variable) => ({
+        label: `$${variable.label ?? variable.name}`,
+        value: `$${variable.name}`,
+        description: `${variable.type} variable`,
```

Comment:
In the clock panel in in twinmaker, we add an icon for the template variable options... not sure if that makes sense here?

See: https://github.com/grafana/clock-panel/pull/108

https://github.com/grafana/clock-panel/pull/108/files#diff-2123f97b7bf9480a61717c843425e483e8d97819592e8c04e4e149eb65b451e8R185

Reply:
@ryantxu 👍 sounds like a good plan to me. updated:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/360020/172256299-b663f4a1-8ca6-4a83-83dd-28bf49eb5b87.png">